modules/configuring-a-multi-node-pytorch-training-job-for-rdma.adoc

```diff
@@ -0,0 +1,291 @@
+
+... Replace `"your container command"` with the command to be run by your training job.
+
+... Set the `*NCCL_SOCKET_IFNAME*` environment variable to specify the IP interface to use for communication.
+
+... [Optional] To explicitly specify the Host Channel Adapter (HCA) that NCCL should use, set the `*NCCL_IB_HCA*` environment variable.
+
+. Specify the base training image name, as follows:
+
+.. Edit the `PyTorchJob` resource YAML code to add the following text:
++
+.Example base training image
+[source,subs="+quotes"]
+----
+image: quay.io/modh/training:py311-cuda121-torch241
+----
+
+.. If you want to use a different base training image, replace the image name accordingly.
++
+For a list of supported training images, see link:https://access.redhat.com/articles/rhoai-supported-configs[{productname-long}: Supported Configurations].
+
+. Specify the requests and limits for the network interface resources.
++
+The name of the resource varies, depending on the NVIDIA Network Operator configuration.
+The resource name might depend on the deployment mode, and is specified in the `NicClusterPolicy` resource.
++
+[NOTE]
+====
+You must use the resource name that matches your configuration.
+The name must correspond to the value advertised by the NVIDIA Network Operator on the cluster nodes.
+====
++
+The following example is for RDMA over Converged Ethernet (RoCE), where the Ethernet RDMA devices are using the RDMA shared device mode.
+
+.. Review the `NicClusterPolicy` resource to identify the `resourceName` value.
++
+.Example NicClusterPolicy
+[source,subs="+quotes"]
+----
+apiVersion: mellanox.com/v1alpha1
+kind: NicClusterPolicy
+spec:
+rdmaSharedDevicePlugin:
+  config: |
+    {
+      "configList": [
+        {
+          "resourceName": "rdma_shared_device_eth",
+          "rdmaHcaMax": 63,
+          "selectors": {
+            "ifNames": ["ens8f0np0"]
+          }
+        }
+      ]
+    }
+----
++
+In this example `NicClusterPolicy` resource, the `resourceName` value is `rdma_shared_device_eth`. 
+
+.. Edit the `PyTorchJob` resource YAML code to add the following text:
++
+.Example requests and limits for the network interface resources
+[source,subs="+quotes"]
+----
+            resources:
+              limits:
+                nvidia.com/gpu: "1"
+                rdma/rdma_shared_device_eth: "1"
+              requests:
+                nvidia.com/gpu: "1"
+                rdma/rdma_shared_device_eth: "1"
+----
+
+.. In the `limits` and `requests` sections, replace the resource name with the resource name from your `NicClusterPolicy` resource (in this example, `rdma_shared_device_eth`).
+
+.. Replace the specified value `1` with the number that you require.
+Ensure that the specified amount is available on your OpenShift cluster.
+
+. Specify the details of the nodes that will run the training job, as follows:
```

Comment:
`Point 7. Specify the details of the nodes that will run the training job, as follows`
I think this section should be removed. Providing the entire example yaml later shows the user that they need to specify master and worker, which they may already know if they have experience with PyTorchJobs - so I think this is enough here.